test/javascripts/fixtures/reactions-topic-fixtues.js.es6

```diff
@@ -0,0 +1,820 @@
re</h3>\n<p>You can configure these Site Settings:</p>\n<ul>\n<li>\n<code>discourse_reactions_enabled</code> To enable the plugin</li>\n<li>\n<code>discourse_reactions_enabled_reactions</code> To define the list of enabled reactions, any emoji is allowed here. <code>default: laughing|open_mouth|cry|angry|thumbsup|thumbsdown</code>\n</li>\n<li>\n<code>discourse_reactions_like_icon</code> To replace the like icon.</li>\n<li>\n<code>discourse_reactions_reaction_for_like</code> To define the name of like reaction.</li>\n</ul>\n<h3>Creating a reaction</h3>\n<p>The reaction can be created either by clicking the like icon OR by hovering and selecting reaction from the reaction picker.</p>\n<p><em>Note:</em></p>\n<ul>\n<li>In Mobile to open the reaction-panel long press on like icon.</li>\n</ul>\n<h3>See who reacted?</h3>\n<p>On the left side of the post, there will be a list of reactions, by clicking on it, there will be a modal displayed with the list of reactions.</p>\n<p><img src="//localhost:3000/images/transparent.png" alt="Screenshot 2021-03-08 at 1.05.19 PM" data-orig-src="upload://dOO4WA7BDmAq5d2EkpRUKDKFq7c" width="1" height="1"></p>\n<p>If there are only likes in the post, a list of reactions will not be available but there will be a like-count besides the like button. clicking on it will display the same modal.</p>\n<p><img src="//localhost:3000/images/transparent.png" alt="Screenshot 2021-03-08 at 1.09.45 PM" data-orig-src="upload://kMxal7tzxL5px35ToK5FBD3Y1Ot" width="1" height="1"></p>\n<p>Don’t want to see this whole modal? Just hover on the reaction-emoji in the list to see the recently reacted users (Available Only on the desktop site).</p>\n<h3>See Your reactions?</h3>\n<p>In the user page activity, there is an extra <code>Reactions</code> tab where you will find all reactions you made.</p>\n<p><img src="//localhost:3000/images/transparent.png" alt="Screenshot 2021-03-08 at 1.34.59 PM" data-orig-src="upload://1NepZH6JY5xZMV3ZqHdisGKwvju" width="1" height="1"></p>\n<h3>See who reacted to your post?</h3>\n<p>In the user page notifications, there is an extra <code>Reactions</code> tab where you will find all reactions made to your posts.</p>\n<p><img src="//localhost:3000/images/transparent.png" alt="Screenshot 2021-03-08 at 1.37.03 PM" data-orig-src="upload://63mpmp96CCpLt3jpGawqlid1XTt" width="1" height="1"></p>\n<h3>Contributors</h3>\n<p><a class="mention" href="/u/awesomerobot">@awesomerobot</a><br>\n<span class="mention">@kris.kotlarek</span><br>\n<span class="mention">@Roman_Rizzi</span></p>\n<h3>Special Thanks</h3>\n<p>Special thanks to <a class="mention" href="/u/awesomerobot">@awesomerobot</a> for being such a nice mentor and for being super helpful in everything <img src="//localhost:3000/images/emoji/twitter/open_hands.png?v=9" title=":open_hands:" class="emoji" alt=":open_hands:"> and <a class="mention-group" href="/groups/team">@team</a> for helping us to make the plugin works perfect by finding bugs as much as possible.</p>',
```

Comment:
Please use a simpler raw, we don't need to have all this text

Reply:
Indeed